src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/Person.kt

```diff
@@ -55,7 +55,10 @@ data class Person(
 
     fun from(offenderDetail: OffenderDetail): Person {
       return Person(
-        otherIdentifiers = OtherIdentifiers(crn = offenderDetail.otherIds.crn),
+        givenName = offenderDetail.firstName,
+        familyName = offenderDetail.surname,
+        dateOfBirth = offenderDetail.dateOfBirth,
+        otherIdentifiers = OtherIdentifiers(crn = offenderDetail.otherIds.crn, pncIdentifier = PNCIdentifier(offenderDetail.otherIds.pncNumber)),
```

Comment:
Similarly 👍 but we should think about naming - we have PncId, PncIdentifer, PncNumber just in this small code sample - they are all the same thing